pkg/data_sources/databricks_credential_acceptance_test.go

```diff
@@ -0,0 +1,68 @@
+package data_sources_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDbtCloudDatabricksCredentialDataSource(t *testing.T) {
+
+	randomProjectName := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+
+	config := databricks_credential(randomProjectName, "moo", "baa", "maa", 64)
+
+	check := resource.ComposeAggregateTestCheckFunc(
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "credential_id"),
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "project_id"),
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "adapter_id"),
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "target_name"),
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "schema"),
+		resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "num_threads"),
+
+		// TODO: revisit when adapters can be created with a service token
+		// as of now, CI is using a spark adapter and doesn't have a catalog
+		// you can uncomment the following line to test locally on a databricks adapter
+
+		// resource.TestCheckResourceAttrSet("data.dbt_cloud_databricks_credential.test", "catalog"),
```

Comment:
There's a flag called TEST_DATABRICKS I used to enable tests for Databricks if it's useful?
Temp fix anyway really

Reply:
Sure, I can update the data source with the env var